net/transmission/Portfile

```diff
@@ -4,10 +4,11 @@ PortSystem      1.0
 PortGroup       github 1.0
 PortGroup       xcode 1.0
 
-github.setup    transmission transmission 2.94
-revision        1
+github.setup    transmission transmission 3.00
+set git_commit  bb6b5a062ee594dfd4b7a12a6b6e860c43849bfd
```

Comment:
Why are you using a explicit commit ? Please change this to use the 3.0.0 release tag.